src/collector/vector_collector.rs

```diff
@@ -0,0 +1,100 @@
+use std::sync::Arc;
+
+
+
+use crate::{DocAddress, SegmentOrdinal, schema::Field, vector::VectorReader};
+
+use super::{Collector, SegmentCollector};
+
+/// Collector for vectors
+///
+/// The collector collects all vectors though all segments.
```

Comment:
What is the point of this collector? Is this for test only?

Reply:
I did this collector to understand better how tantivy works. A TopDocs collector would indeed suffice. I will remove it from the PR.